### PR TITLE
add very simple getting started for mesos

### LINKED ISSUE #854: Write mesos/swarm getting started docs
/cc @tnachen @jimenez @vieux

### PR DESCRIPTION
**Pull request by vieux**
close #854

### CODE REVIEW

**Comment by vieux** [comment]
@jimenez @tnachen can you take a look

@moxiegirl can you make this proper english and pretty :D

File: cluster/mesos/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Using Swarm and Mesos
```
Comment (moxiegirl) [inline cluster/mesos/README.md]: # Using Docker Swarm and Mesos

Swarm comes with a built-in scheduler that works with the swarm manager to schedule container resources. You can completly replace the built-in scheduler with a 3rd party scheduler. For example, you can replace it with the Mesos scheduler as described here.

When using Docker Swarm and Mesos, you use the Docker client to ask the swarm
manager to schedule containers. The swarm manager then schedules those
containers on a Mesos cluster.

## Prerequisites

Each node in your swarm must run a Mesos slave. The slave must be capable of starting tasks in a Docker Container using the `--containerizer=docker` option.

You need to configure two TCP ports on the slave. One port to listen for the swarm manager, for example 2375. And a second TCP port to listen for the Mesos master, for example 3375.

## Start the Docker Swarm manager

If you use a single Mesos master:

```     
docker run -d -p <swarm_port>:2375 -p 3375:3375 swarm manage -c mesos-experimental --cluster-opt mesos.address=<public_machine_ip> --cluster-opt mesos.port=3375 <mesos_master_ip>:<mesos_master:port>
```

If you use multiple Mesos masters:

```
docker run -d -p <swarm_port>:2375 -p 3375:3375 swarm manage -c mesos-experimental --cluster-opt mesos.address=<public_machine_ip> --cluster-opt mesos.port=3375 zk://<mesos_masters_url>
```

Once the manager is running, check your configuration by running `docker info` as follows:

```
docker -H tcp://<manager_ip:manager_port> info
```

For example, if you run the manager locally on your machine:

```
Containers: 0
Offers: 2
  Offer: 20150609-222929-1327399946-5050-14390-O6286
     └ cpus: 2
     └ mem: 1006 MiB
     └ disk: 34.37 GiB
     └ ports: 31000-32000
  Offer: 20150609-222929-1327399946-5050-14390-O6287
     └ cpus: 2
     └ mem: 1006 MiB
     └ disk: 34.37 GiB
     └ ports: 31000-32000
```

File: cluster/mesos/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Docker Swarm comes with a builtin scheduler that you can completly replace by a 3rd party scheduler.
+You can for example use Mesos to schedule containers.
```
Comment (chanwit) [inline cluster/mesos/README.md]: You can, for example, use Mesos to schedule containers.

File: cluster/mesos/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+You will use the regular Docker client to ask the swarm manager to schedule containers, and the
```
Comment (chanwit) [inline cluster/mesos/README.md]: the Swarm manager

File: cluster/mesos/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+swarm manager will schedule those containers on a Mesos cluster.
```
Comment (chanwit) [inline cluster/mesos/README.md]: Swarm manager

File: cluster/mesos/README.md (hunk 1 above)
Comment (chanwit) [inline cluster/mesos/README.md]: I think my sparse comments are not needed any more :)

**Comment by vieux** [comment]
@moxiegirl PTAL

**Comment by moxiegirl** [comment]
LGTM

**Comment by jimenez** [comment]
LGTM

**Pull request merged**